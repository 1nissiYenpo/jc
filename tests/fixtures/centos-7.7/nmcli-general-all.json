[{"running":"running","version":"1.18.0","state":"connected","startup":"started","connectivity":"full","networking":"enabled","wifi_hw":"enabled","wifi":"enabled","wwan_hw":"enabled","wwan":"enabled"}]
