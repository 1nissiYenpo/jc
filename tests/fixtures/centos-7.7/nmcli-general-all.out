RUNNING  VERSION  STATE      STARTUP  CONNECTIVITY  NETWORKING  WIFI-HW  WIFI     WWAN-HW  WWAN    
running  1.18.0   connected  started  full          enabled     enabled  enabled  enabled  enabled 

